plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -67,34 +67,40 @@ func (c *CachedNodeInfo) GetNodeInfo(id string) (*api.Node, error) {
 	return node.(*api.Node), nil
 }
 
+/**
+*  Note that predicateMetdata and matchingPodAntiAffinityTerm need to be declared in the same file
+*  due to the way declarations are processed in predicate declaration unit tests.
+ */
```

Comment:
We are generally trying to avoid such style comments. Please change to multiple "// coment" lines.